di_website/templates/includes/scaffold/header-assets.html

```diff
@@ -10,3 +10,4 @@
 <script src="{% static 'js/libs/modernizr.js' %}"></script>
 {% include 'includes/scaffold/favicon.html' %}
 <link href="{% static 'css/di_website.min.css' %}" rel="stylesheet" />
+<link href="{% static 'publications/css/style.css' %}" rel="stylesheet" />
```

Comment:
Nope ... this is shared across the entire app, so why would we need publications CSS for non-publications pages? Unless u added the classes in the wrong place. In which case, add them to the global CSS file `di_website.css` ... you'll need to minify it and update the minified version of the file ... this isn't tied to a task runner so u'll need an online minifier ... shouldn't be a problem.